bin/cylc-insert

```diff
@@ -43,39 +43,47 @@ if '--use-ssh' in sys.argv[1:]:
 import cylc.flags
 from cylc.prompt import prompt
 from cylc.network.suite_command import SuiteCommandClient
-from cylc.CylcOptionParsers import cop, multitask_usage
+from cylc.CylcOptionParsers import cop
 
 
 def main():
     parser = cop(
-        __doc__ + multitask_usage,
-        pyro=True, multitask=True,
+        __doc__, pyro=True, multitask=True,
         argdoc=[
             ("REG", "Suite name"),
-            ('MATCH', 'Task or family name matching regular expression'),
-            ('CYCLE_POINT', 'Cycle point (e.g. date-time or integer)'),
-            ('[STOP_POINT]', 'Optional stop cycle point for inserted task.')])
```

Comment:
So, does the removal of STOP_POINT now mean that I'm unable to insert a task with a view to running it between to earlier cylce points? (i.e. re-running an earlier tasks between two dates)